src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -14,12 +16,12 @@ library ConstantsLib {
     /// @notice Target utilization (scaled by WAD).
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
-    /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    /// @notice Initial rate at target per second (scaled by WAD) (4% APR <=> 1% min APR).
+    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.01 ether) * CURVE_STEEPNESS / WAD_INT / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Minimum rate at target per second (scaled by WAD) (0.1% APR <=> 0.025% min APR).
+    int256 public constant MIN_RATE_AT_TARGET = int256(0.025 * 0.01 ether) * CURVE_STEEPNESS / WAD_INT / 365 days;
 
-    /// @notice Maximum rate at target per second (scaled by WAD) (1B% APR).
-    int256 public constant MAX_RATE_AT_TARGET = int256(0.01e9 ether) / 365 days;
+    /// @notice Maximum rate at target per second (scaled by WAD) (200% APR <=> 800% max APR).
+    int256 public constant MAX_RATE_AT_TARGET = int256(800 * 0.01 ether) * WAD_INT / CURVE_STEEPNESS / 365 days;
```

Comment:
I'm not sure that this is clearer (btw the * WAD_INT was useless)
```suggestion
    /// @notice Maximum rate at target per second (scaled by WAD) (maximum rate equals MAX_RATE_AT_TARGET * CURVE_STEEPNESS).
    int256 public constant MAX_RATE_AT_TARGET = int256(200 * 0.01 ether) / 365 days;
```

Reply:
> btw the * WAD_INT was useless

why? the scale of `CURVE_STEEPNESS` is wad

```suggestion
    /// @notice Maximum rate at target per second (scaled by WAD) (200% APR <=> 800% max APR).
    int256 public constant MAX_RATE_AT_TARGET = int256(200 * 0.01 ether) / 365 days;
```

Reply:
> why? the scale of CURVE_STEEPNESS is wad

yes sorry you're right